machine/scripts/db.d/10-mysqld.sh

```diff
@@ -28,6 +28,7 @@ yum install -y mysql-server
 service mysqld start >> $BOOTSTRAP_LOG 2>&1
 mysql -uroot -e "
     GRANT ALL PRIVILEGES ON *.* TO 'root'@'localhost' WITH GRANT OPTION;
+    GRANT ALL PRIVILEGES ON *.* TO 'root'@'127.0.0.1' WITH GRANT OPTION;
```

Comment:
@davidalger Since we will be creating MySQL users with specific permissions to the database(s) we create, do you think I should remove this line and we can just grant this privilege to the new MySQL users that we create?